SonarQube.TeamBuild.PreProcessor/SonarWebService.cs

```diff
@@ -256,6 +257,21 @@ public bool TryDownloadEmbeddedFile(string pluginKey, string embeddedFileName, s
 
         #region Private methods
 
+        private string addOrganization(string encodedUrl, string organization)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename method 'addOrganization' to match camel case naming rules, consider using 'AddOrganization'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS100)